src/components/dashboard/jumbotron/CurrentJumbotron.js

```diff
@@ -0,0 +1,84 @@
+import React, { Component } from 'react';
+
+/**
+ *  CurrentJumbotron displays information for a trip
+ *  that is currently in progress
+ */
+
+export default class CurrentJumbotron extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
```

Comment:
See comment earlier about the use of state vs helper functions. I think this could be applied here too.